frontend/src/components/ExperimentsTable.jsx

```diff
@@ -43,6 +43,11 @@ const ExperimentsTable = (props) => {
       onResultsConfigSelectUpdate(project.id, resultId, !evt.target.checked);
     });
   };
+  const handleGroupedResultsConfigSelectChange = (groupedResultKeys) => (evt) => {
```

Comment:
This method name is ungood. It returns event handler, so I would name `generateHandleGroupedResultsConfigSelectChange`.

Reply:
fixed, thx